### PR TITLE
Data collection

### PR DESCRIPTION
**Pull request by Xierumeng**

### CODE REVIEW

**Review by Trotyl15 — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -5,3 +5,7 @@ Using Ultralytics YOLOv8.
 ## Training
 
 Follow the [instructions](https://uwarg-docs.atlassian.net/l/cp/XBXNaCnx).
+
+## Imaging
+
+Follow the [instructions](https://uwarg-docs.atlassian.net/l/cp/JuBLUpJr).
```
Comment (mgupta27) [inline README.md:11]: Are you planning on making another PR adding to the `README` with instructions on setup?

**Review by mgupta27 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Xierumeng) [reply README.md:11]: That Confluence page will contain instructions. I will hold off on merging until they are written.

**Pull request merged**